### PR TITLE
fix: Badge size mismatch

### LINKED ISSUE #586: Cliparts size is 40x11 even though badge size is 44x11
First up, thanks for your amazing work on this app. Really love the easy way to change badges!
**Describe the bug**
I noticed that cliparts can only be created in a 40x11 px format (stored png also has this format). But the badge has a 44x11 px width. Any reason for that?

**To Reproduce**
Create a new clipart and fill it with pixels. Now go to "create badge" and insert it. There will be an area of 2px left and right of the clipart which will not have any red pixels.

**Expected behavior**
I would expect the canvas for new cliparts to be 44px wide.

**Screenshots**
Photo of the clipart:
photo5208504173140749201
Photo of the badge with padding.
photo5208648853409082519

**Smartphone Info:**
|Device         | Samsung Galaxy S5 (klte) |
|Android Version| 9 (Lineageos)    |
App version: 1.5.0

**Would you like to work on the issue?**
My android skills are low, so rather not.

### PR DESCRIPTION
**Pull request by akri16**
Fixes #586

Changes:
1. Change everything to 44 x 11
2. Fix related errors and refactor

Screenshots for the change:

**Creating the clipart** 

WhatsApp Image 2021-01-08 at 16 38 37 (1)

**Static view**

WhatsApp Image 2021-01-08 at 16 38 37

**Animated view**

https://user-images.githubusercontent.com/54491362/104010632-d137a900-51d2-11eb-94c2-d7005847e3a2.mp4

**Normal Cliparts**

WhatsApp Image 2021-01-08 at 16 45 05

### CODE REVIEW

File: android/src/main/java/org/fossasia/badgemagic/ui/custom/PreviewBadge.kt [hunk 1]
```diff
@@ -21,6 +21,7 @@ import org.fossasia.badgemagic.data.Speed
 import org.fossasia.badgemagic.data.badge_preview.Cell
 import org.fossasia.badgemagic.data.badge_preview.CheckList
 import org.fossasia.badgemagic.util.Converters.hexToBin
+import timber.log.Timber
```
Comment (adityastic) [inline android/src/main/java/org/fossasia/badgemagic/ui/custom/PreviewBadge.kt]: This file can be rolled back

File: android/src/main/java/org/fossasia/badgemagic/util/Converters.kt [hunk 2]
```diff
@@ -39,16 +40,18 @@ object Converters {
...
-            bm = ImageUtils.scaleBitmap((drawableIcon).bitmap, 40)
+            bm = ImageUtils.scaleBitmap((drawableIcon).bitmap, 44)
         return convertBitmapToLEDHex(bm, invertLED)
     }
 
     fun convertBitmapToLEDHex(bm: Bitmap, invertLED: Boolean): List<String> {
         val height = bm.height
         val width = bm.width
 
+        Timber.d("$height $width")
```
Comment (adityastic) [inline android/src/main/java/org/fossasia/badgemagic/util/Converters.kt]: Logging, can be removed

File: android/src/main/res/values/integers.xml [hunk 3]
```diff
@@ -1,4 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <integer name="knob_indicator_width">10</integer>
+    <integer name="clip_viewport_width">44</integer>
```
Comment (adityastic) [inline android/src/main/res/values/integers.xml]: Not sure if this is used

File: android/src/main/java/org/fossasia/badgemagic/util/Converters.kt [hunk 4]
```diff
@@ -13,6 +13,7 @@ import kotlin.math.ceil
 import kotlin.math.floor
 import org.fossasia.badgemagic.data.badge_preview.CheckList
 import org.fossasia.badgemagic.device.DataToByteArrayConverter
+import timber.log.Timber
```
Comment (adityastic) [inline android/src/main/java/org/fossasia/badgemagic/util/Converters.kt]: Same, timber. Logging can be removed

File: android/src/main/java/org/fossasia/badgemagic/ui/custom/PreviewBadge.kt [hunk 5]
```diff
@@ -494,9 +495,12 @@ class PreviewBadge : View {
             }
         }
 
+        Timber.d("$allHex")
```
Comment (adityastic) [inline android/src/main/java/org/fossasia/badgemagic/ui/custom/PreviewBadge.kt]: Can be rolled back

**Review by adityastic — CHANGES_REQUESTED** [review]

**Comment by akri16** [comment]
I`ve made the requested changes and force pushed it after amending the commit. But I'm unable to 'Resolve Conversation' these requests. Is there a cleaner way to do the same?

**Review by adityastic — APPROVED** [review]
Lgtm

**Comment by adityastic** [comment]
> Is there a cleaner way to do the same?
>
Once the new changes are detected. The comments on the last changes automatically get resolved

**Pull request merged**